test/unit/transpiler/passes/scheduling/test_dynamical_decoupling.py

```diff
@@ -569,14 +549,16 @@ def test_back_to_back_c_if(self):
         expected.delay(800, 1)
         expected.delay(800, 2)
         expected.barrier()
-        expected.x(1).c_if(0, True)
-        expected.x(2).c_if(0, True)
+        with expected.if_test((0, True)):
+            expected.delay(50, 0)
+            expected.x(1)
+            expected.delay(50, 2)
+        with expected.if_test((0, True)):
+            expected.delay(50, 0)
+            expected.delay(50, 1)
+            expected.x(2)
         expected.barrier()
-        expected.delay(225, 0)
-        expected.x(0)
-        expected.delay(450, 0)
-        expected.x(0)
-        expected.delay(225, 0)
+        expected.delay(1000, 0)
```

Comment:
This is because the qubit has not yet been touched., ie., it is assumed to be in the `|0>` state. This may be partially controlled with `skip_reset_qubits` and is behaviour inherited from the standard Qiskit implementation.